package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-nuxt-app",
-  "version": "2.6.0",
+  "version": "2.6.1",
```

Comment:
The version doesn’t need to be bumped, we will bump it before release

Reply:
@clarkdo 
Sorry, I fixed it now.
https://github.com/nuxt/create-nuxt-app/pull/207/commits/3839792f6c7f4e45e30e923d92ec8280c2841e77